src/adagrams.js

```diff
@@ -1,15 +1,121 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const letters = [
```

Comment:
since letters is a constant variable it should be spelled with all caps like LETTERS